cmd/nodecmd/create.go

```diff
@@ -240,21 +271,45 @@ func stringToAWSVolumeType(input string) types.VolumeType {
 }
 
 func createNodes(cmd *cobra.Command, args []string) error {
+	var err error
 	clusterName := args[0]
+	network := models.UndefinedNetwork
 	if err := preCreateChecks(clusterName); err != nil {
 		return err
 	}
-	network, err := networkoptions.GetNetworkFromCmdLineFlags(
-		app,
-		"",
-		globalNetworkFlags,
-		false,
-		true,
-		createSupportedNetworkOptions,
-		"",
-	)
-	if err != nil {
-		return err
+	// etna devnet constants
+	if useEtnaDevnet {
```

Comment:
It will also be cleaner here to support the Etna case within `GetNetworkFromCmdLineFlags` rather than needing to have custom handling for it throughout.

Reply:
etnadevnet is shiortcut with predefined bootstrap configuration for custom external network.
let me think about appropriate name for this. External or custom might be a correct name.
Once it's merged we should be able to support any other remote networks like etnadevnet or whatever we will need going forward